src/pages/commonPage.ts

```diff
@@ -217,7 +217,7 @@ export default class CommonPage implements CommonPageInterface {
   ): Promise<Page> {
     const [newPage] = await Promise.all([
       page.waitForEvent('popup'),
-      page.locator(selector).click(),
+      page.locator(selector).first().click(),
```

Comment:
```suggestion
      page.locator(selector).first().click(),
```

now you can revert this one ;)